environment/src/test/java/com/sequenceiq/environment/environment/service/EnvironmentCreationServiceTest.java

```diff
@@ -480,6 +480,41 @@ void testParameterVerificationError() {
         verify(reactorFlowManager, never()).triggerCreationFlow(anyLong(), eq(ENVIRONMENT_NAME), eq(USER), anyString());
     }
 
+    @Test
+    void testEncryptionKeyUrlValidationError() {
+        final EnvironmentCreationDto environmentCreationDto = EnvironmentCreationDto.builder()
+                .withName(ENVIRONMENT_NAME)
+                .withCloudPlatform("AZURE")
+                .withCreator(CRN)
+                .withAccountId(ACCOUNT_ID)
+                .withAuthentication(AuthenticationDto.builder().build())
+                .build();
+        final Environment environment = new Environment();
+        environment.setName(ENVIRONMENT_NAME);
+        environment.setId(1L);
+        environment.setAccountId(ACCOUNT_ID);
+        Credential credential = new Credential();
+        credential.setCloudPlatform("AZURE");
+
+        ValidationResultBuilder validationResultBuilder = new ValidationResultBuilder();
+        validationResultBuilder.error("error");
+        when(validatorService.validateEncryptionKeyUrl(any())).thenReturn(validationResultBuilder.build());
+
+        when(environmentService.isNameOccupied(eq(ENVIRONMENT_NAME), eq(ACCOUNT_ID))).thenReturn(false);
+        when(environmentDtoConverter.creationDtoToEnvironment(eq(environmentCreationDto))).thenReturn(environment);
+        when(environmentResourceService.getCredentialFromRequest(any(), any())).thenReturn(credential);
+        when(validatorService.validateParentChildRelation(any(), any())).thenReturn(ValidationResult.builder().build());
+        when(validatorService.validateNetworkCreation(any(), any())).thenReturn(ValidationResult.builder());
+        when(validatorService.validateFreeIpaCreation(any())).thenReturn(ValidationResult.builder().build());
+        when(authenticationDtoConverter.dtoToAuthentication(any())).thenReturn(new EnvironmentAuthentication());
+        when(environmentService.save(any())).thenReturn(environment);
+        when(entitlementService.azureEnabled(eq(ACCOUNT_ID))).thenReturn(true);
+        when(entitlementService.isAzureDiskSSEWithCMKEnabled(eq(ACCOUNT_ID))).thenReturn(true);
```

Comment:
This expectation is now unnecessary; the `isAzureDiskSSEWithCMKEnabled()` check is a part of `EnvironmentValidatorService.validateEncryptionKeyUrl()`.